server/src/routes/api/v1/podcastVotesRouter.js

```diff
@@ -0,0 +1,105 @@
+import express from 'express'
+import objection from 'objection'
+const { ValidationError } = objection
+
+import { Vote } from '../../../models/index.js'
+
+const podcastVotesRouter = new express.Router({ mergeParams: true })
+
+podcastVotesRouter.get('/', async (req, res) => {
+  const { podcastId } = req.params
+  let userId
+  if (req.user) userId = req.user.id
+
+  try {
+    let votes = {}
+    votes.userVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .andWhere('userId', userId)
+      .first()
```

Comment:
these where statements are unnecessary. Any time you have to call `first()` at the end of a query, you likely want to be using `findOne` instead of `.where`